test/unittests/testInit.h

```diff
@@ -43,6 +45,8 @@
 #define RES_CER_STATUS(i)       res.contextElementResponseVector.get(i)->statusCode
 #define RES_CER_ATTR(i, j)      res.contextElementResponseVector.get(i)->contextElement.contextAttributeVector.get(j)
 
+#define C_STR_FIELD(b, f)       getStringField(b, f).c_str();
```

Comment:
Although removed from the non-test code base, the macro still have sense in the unit test code.

NTC